doc/conf.py

```diff
@@ -292,11 +292,11 @@ class TypeMock(type):
     "table_header_border": "#19b37b",
     "download_button": "#19b37b",
     # gallery options
-    # "github_repo": "XanaduAI/PennyLane",
+    # "github_repo": "PennyLaneAI/PennyLane",
```

Comment:
Those are commented lines, should it be removed?

Reply:
Not sure, I know Antal had it changed aswell ..? Or is it a temporary comment for some reason?